src/traces/surface/defaults.js

```diff
@@ -50,13 +50,19 @@ module.exports = function supplyDefaults(traceIn, traceOut, defaultColor, layout
         }
     }
 
-    coerce('lighting.ambient');
-    coerce('lighting.diffuse');
-    coerce('lighting.specular');
-    coerce('lighting.roughness');
-    coerce('lighting.fresnel');
-    coerce('hidesurface');
-    coerce('opacity');
+    //Coerce remaining properties
+    [
+        'lighting.ambient',
+        'lighting.diffuse',
+        'lighting.specular',
+        'lighting.roughness',
+        'lighting.fresnel',
+        'lightposition.x',
+        'lightposition.y',
+        'lightposition.z',
+        'hidesurface',
```

Comment:
Correct, initially I wasn't going to shoot for covering the `surface3d` case, just `mesh3d`; the reason for adding the properties there is that `mesh3d` piggybacks on it and we may at some point need this feature for the surface anyway. I'd suggest let's keep this PR for the `mesh3d` as it's being expected today and I'd do a subsequent small PR for the `surface3d` case.